docs_user/assemblies/assembly_preparing-the-block-storage-service-for-adoption.adoc

```diff
@@ -43,27 +32,15 @@ spec:
             [ceph]
 < . . . >
 ----
-
-Reminder that for volume backends that have sensitive information using `Secret`
-and the `customServiceConfigSecrets` key is the recommended way to go.
-
-For adoption instead of using a whole deployment manifest you use a targeted
-patch, like you did with other services, and in this patch you will enable the
-different Block Storage services with their specific configurations.
-
++
 [WARNING]
-Check that all configuration options are still valid for the new
-{rhos_prev_long} version. Configuration options may have been deprecated,
-removed, or added. This applies to both backend driver specific configuration
-options and other generic options.
+Check that all configuration options are still valid for the new {rhos_long} version. Configuration options might be deprecated, removed, or added. This applies to both back-end driver-specific configuration options and other generic options.
 
 ifeval::["{build}" != "downstream"]
 There are 2 ways to prepare a {block_storage} configuration for adoption, tailor-making
```

Comment:
Suggest rephrasing, e.g. There are two ways to prepare a {block_storage} configuration for adoption. You can customize the configuration or prepare a quick configuration. There is no difference in how {block_storage} operates with both methods, but customization is recommended whenever possible.